18_regexp_lab/02.py

```diff
@@ -1,6 +1,7 @@
-"""
-Write a function toCamelCase(text) that returns the text camel cased
-Write a function to_underscore(text) that returns the text with 
-underscores between words
-"""
-
+import re
+
+def toCamelCase(underscore_line):

```

Comment:
Function names in python are written in lowercase (same as variable), so use `to_camelcase`